front/src/Apps/Dashboard/Components/BsdCard/BsdCard.tsx

```diff
@@ -207,7 +207,7 @@ function BsdCard({
   // - we are in the "Collected" tab and there is a number plate
   const displayTransporterNumberPlate =
     !!currentTransporterInfos &&
-    currentTransporterInfos.transporterMode === TransportMode.Road &&
+    // currentTransporterInfos.transporterMode === TransportMode.Road &&
```

Comment:
Est-ce qu'on ne serait pas un poil plus restrictif ici : 
```suggestion
    currentTransporterInfos.transporterMode === TransportMode.Road 
    // permet de gérer un trou dans la raquette en terme de validation des données
    // qui ne rend pas le mode de transport obligatoire à la signature transporteur
    // en attente de correction Cf ticket tra-xxxx
    || currentTransporterInfos.transporterMode === null
```